test-utils/src/main/json/restate_config_schema.json

```diff
@@ -157,11 +152,24 @@
     },
     "bifrost": {
       "default": {
-        "default_provider": "File",
+        "default_provider": "Local",
         "providers_config": {
-          "File": {
-            "path": "target/logs/"
-          }
+          "Local": {
+            "flush_wal_on_commit": true,
+            "rocksdb_cache_size": 0,
+            "rocksdb_disable_statistics": true,
+            "rocksdb_disable_wal": false,
+            "rocksdb_max_total_wal_size": 2147483648,
+            "rocksdb_threads": 10,
+            "rocksdb_write_buffer_size": 0,
+            "writer_commit_batch_size_threshold": 200,
+            "writer_commit_time_interval": {
+              "nanos": 13000000,
+              "secs": 0
+            },
+            "writer_queue_len": 200
+          },
+          "Memory": null
```

Comment:
What does e2e use this schema for?

Reply:
It doesn't use it as far as I know. I've basically copied the generated schema over from the restate repository (generated via `cargo xtask generate-config-schema`).